extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -154,6 +157,23 @@ void build(CombinedIndexBuildItem index,
         }
     }
 
+    @BuildStep
+    ValidationPhaseBuildItem.ValidationErrorBuildItem validate(ValidationPhaseBuildItem validationPhase,
+            CombinedIndexBuildItem index) throws BuildException {
```

Comment:
```suggestion
            CombinedIndexBuildItem index)  {
```
We do not throw any exception

Reply:
It can be thrown by `io.quarkus.panache.common.deployment.JandexUtil.isSubclassOf` that delcares it as a checked exception.

Reply:
Ah I might have missed it, sorry.